pkg/plugins/policies/meshhttproute/plugin/v1alpha1/plugin_test.go

```diff
@@ -0,0 +1,411 @@
+								},
+								Rules: []api.Rule{{
+									Matches: []api.Match{{
+										Path: api.PathMatch{
+											Prefix: "/v1",
+										},
+									}},
+									Default: api.RuleConf{
+										BackendRefs: &[]api.BackendRef{{
+											TargetRef: common_api.TargetRef{
+												Kind: common_api.MeshService,
+												Name: "backend",
+											},
+											Weight: 100,
+										}},
+									},
+								}},
+							}},
+						},
+					}, {
+						Meta: &test_model.ResourceMeta{
+							Mesh: core_model.DefaultMesh,
+							Name: "route-2",
+						},
+						Spec: &api.MeshHTTPRoute{
+							TargetRef: common_api.TargetRef{
+								Kind: common_api.MeshService,
+								Name: "frontend",
+							},
+							To: []api.To{{
+								TargetRef: common_api.TargetRef{
+									Kind: common_api.MeshService,
+									Name: "backend",
+								},
+								Rules: []api.Rule{{
+									Matches: []api.Match{{
+										Path: api.PathMatch{
+											Prefix: "/v1",
+										},
+									}},
+									Default: api.RuleConf{
+										BackendRefs: &[]api.BackendRef{{
+											TargetRef: common_api.TargetRef{
+												Kind: common_api.MeshService,
+												Name: "backend",
+											},
+											Weight: 100,
+										}},
+									},
+								}},
+							}},
+						},
+					}},
+				},
+			},
+		},
+		expectedRoutes: core_xds.ToRules{
+			Rules: core_xds.Rules{
+				{
+					Subset: core_xds.MeshService("backend"),
+					Conf: api.PolicyDefault{
+						AppendRules: []api.Rule{
+							{
+								Matches: []api.Match{{
+									Path: api.PathMatch{
+										Prefix: "/v1",
+									},
+								}},
+								Default: api.RuleConf{
+									BackendRefs: &[]api.BackendRef{{
+										TargetRef: common_api.TargetRef{
+											Kind: common_api.MeshService,
+											Name: "backend",
+										},
+										Weight: 100,
+									}},
+								},
+							},
+							{
+								Matches: []api.Match{{
+									Path: api.PathMatch{
+										Prefix: "/v1",
+									},
+								}},
+								Default: api.RuleConf{
+									BackendRefs: &[]api.BackendRef{{
+										TargetRef: common_api.TargetRef{
+											Kind: common_api.MeshService,
+											Name: "backend",
+										},
+										Weight: 100,
+									}},
+								},
+							},
+						},
+					},
+					Origin: []core_model.ResourceMeta{
+						&test_model.ResourceMeta{
+							Mesh: "default",
+							Name: "route-1",
+						},
+						&test_model.ResourceMeta{
+							Mesh: "default",
+							Name: "route-2",
+						},
+					},
+				},
+			},
+		},
+	}),
+	)
+	type routesTestCase struct {
+		rules          []plugin.ToRouteRule
+		serviceName    string
+		expectedRoutes []plugin.Route
+	}
+	DescribeTable("FindRoutes", func(given routesTestCase) {
+		routes := plugin.FindRoutes(given.rules, given.serviceName)
+		Expect(routes).To(Equal(given.expectedRoutes))
+	}, Entry("basic", routesTestCase{
+		rules: []plugin.ToRouteRule{{
```

Comment:
I don't know, for some reason I settled on the first. Probably because the indentation got so bad somewhere. Why can't gofmt just be stricter... If we use [gofumpt](https://github.com/mvdan/gofumpt) it'd pick the second way.

Reply:
@jakubdyszkiewicz I'm going to resolve this because IMO we shouldn't be policing `gofmt`ed code in reviews. If there are problems with `gofmt` accepted code, we should switch to something like `gofumpt`